### DISCUSSION TITLE
Should we be logging receipt of events and http requests out to the DB?

### DISCUSSION THREAD

**Discussion by samcorky** [somakeit/smib#175] [Ideas]
Currently the process logs out that we have received an http event, or if we have received an unhanded event... not any further details about them really. 

Should this be made more consistent and possibly log all requests and their payload to the DB? Housekeeping scheduled task (in the app) can be created to clear any requests older than x days using the `@app.schedule()` decorator hook

**Comment by sjefferson99** [comment]
Has there been any need to date to see a log of events? If we don't need it, don't build, until we do, then build it.

**Reply by samcorky** [reply]
No need as of yet. Just thought I'd dump some ideas here
